examples/imagenet/train_imagenet.py

```diff
@@ -27,6 +28,15 @@
     import models_v2.nin as nin
     import models_v2.resnet50 as resnet50
 
+# Check Python version if it supports multiprocessing.set_start_method,
+# which was introduced in Python 3.4
+major, minor, _, _, _ = sys.version_info
+if major <= 2 or (major >= 3 and minor < 4):
+    sys.stderr.write("Error: ImageNet example uses chainer.iterators."
+                     "MultiprocessIterator,\n"
+                     "which works only PYthon >= 3.4.")
```

Comment:
Could you add link to this URL (http://chainermn.readthedocs.io/en/master/tutorial/tips_faqs.html#using-multiprocessiterator)? e.g., https://github.com/cupy/cupy/blob/master/cupy_setup_build.py#L350-L351